marmelroy/PhoneNumberKit
# Rename class `PhoneNumberKit` to `PhoneNumberUtility`

Fixes #755 .

This renames the class `PhoneNumberKit` to `PhoneNumberUtility` to resolve the issue described in #755 . 

This is a breaking API change, which probably requires a major version bump. See how the sample project needed to be adapted: examples/PhoneBook/Sample/ViewController.swift

## Testing Steps:
* build an artifact (xcframework or .framework) with `BUILD_LIBRARY_FOR_DISTRIBUTION=YES`
* Import the artifact into another project
* make sure Xcode is not reporting errors like this anymore `'PhoneNumber' is not a member type of class 'PhoneNumberKit.PhoneNumberKit'`
